apps/designer/app/routes/dashboard/index.tsx

```diff
@@ -31,7 +31,7 @@ export const action: ActionFunction = async ({ request }) => {
   const { project: title } = schema.parse(await request.formData());
 
   const { userId, headers } = await ensureUserCookie(request);
-  const authenticatedUser = await authenticator.isAuthenticated(request);
+  const authenticatedUser = await findAuthenticatedUser(request);
```

Comment:
And later even if user is null we create project anyway?

Reply:
The action will fail with db constraint. Though it will not be called this way caz 
 dashboard redirects to login page.

Reply:
if db fails then types are wrong